gradle/libs.versions.toml

```diff
@@ -28,6 +28,10 @@ swagger = "1.6.14"
 
 
 [libraries]
+#
+# If a dependency is removed, check whether the LICENSE and/or NOTICE files need to be adopted
```

Comment:
Couldn't we just automatically check whether or not our license file matches our dependencies ? We could always do this in a future PR